src/components/molecules/ArtifactsList/ArtifactsList.tsx

```diff
@@ -15,14 +15,24 @@ import {ArtifactsListContainer, ArtifactsListItem, StyledDownloadIcon, StyledSpa
 type ArtifactsListProps = {
   artifacts: Artifact[];
   testExecutionId: string;
+  isLoading?: boolean;
 };
 
 const ArtifactsList: React.FC<ArtifactsListProps> = props => {
-  const {artifacts, testExecutionId} = props;
+  const {artifacts, testExecutionId, isLoading} = props;
 
   const downloadArtifact = downloadFile(testExecutionId);
 
   const renderedArtifactsList = useMemo(() => {
+    if (isLoading) {
+      return (
+        <>
+          <Skeleton additionalStyles={{lineHeight: 40, color: Colors.slate900, contrastColor: Colors.slate700}} />
+          <Skeleton additionalStyles={{lineHeight: 40, color: Colors.slate900, contrastColor: Colors.slate700}} />
```

Comment:
Maybe reuse these styles?

Reply:
Good point, but I will keep it for now this way until the OSS/Cloud alignment will be finished, to avoid unnecessary PRs